### ISSUE TITLE
Returns Null

### ISSUE THREAD

**Issue by jonjie0317** [greggilbert/recaptcha#132]
It returns `null` when I used `dd()` in it. 
**My code:**

`dd($request->input('g-recaptcha-response'));`

**Comment by phpspider** [comment]
Try with dd($request->all());

**Comment by jonjie0317** [comment]
it also returns null, because we're just returning all inputs. any idea?  need help, Thanks!

**Comment by phpspider** [comment]
Share your method and html form if possible.

**Comment by jonjie0317** [comment]
**form**
```
<form action="{{ route('auth.form') }}" method="post">
{!! Recaptcha::render() !!}
<input type="submit" value="Submit">
</form>
```

**method**

```
public function postForm(Request $request){
     dd($request->input('g-recaptcha-response'));

     // Some private code...
} 
```

As simple as that, just for testing, but doesnt work.

**Comment by phpspider** [comment]
Amazing. Same code is working fine at my end. Also working fine dd($request->all());

**Comment by m1guelpf** [comment]
@jonjie0317 Maybe you should first validate that that field isn't empty...

**Comment by greggilbert** [comment]
@jonjie0317 Add `dd($request->all());` at the top of `postForm` (you can also do `dd(\Request::all())`). Also, are you type sniffing the `Request` properly? Make sure that the `$request` object you're looking at is of type `Illuminate\Http\Request`.

**Comment by jonjie0317** [comment]
Yes, I typed it properly `Illuminate\Http\Request`. I tried `dd($request->all());`, it returns:

array:11 [▼
  "username" => ""
  "password" => ""
  "_token" => "75rpLAPo6gXdsP9FLUWA6ma3pvcT4db6agE1yxaRw"
]

**Comment by ivzinoviev** [comment]
Same issue.
If form sended from Firefox always
`"g-recaptcha-response" => null`
In Chrome everything working fine.
App log clear, no errors in Apache too.
Set config
`'curl_verify' => false,`
and tryed
`'driver'      => 'curl',`
But response still `null`

**Comment by jonjie0317** [comment]
@ivzinoviev  Have you try to change the place of your `{!! Recaptcha::render() !!}` in the form?  You may try at the beginning of the form. e.g. <br>
```
<form action="" method="post">
{!! Recaptcha::render() !!}

<!-- Other inputs here... -->
<input type="submit" value="Register">
</form>
```

**Comment by ivzinoviev** [comment]
@jonjie0317  Yes, it takes no effect.
Only founded solution is using **https** instead http.

**Comment by jonjie0317** [comment]
@ivzinoviev  Oh, that's great! 👍 thank you for sharing your solution, so this will help others.

**Comment by v1shky** [comment]
I am on HTTPS Still face the issue.

Any help?

**Comment by IslamEso** [comment]
I replaced the 
{!! Recaptcha::render() !!}
with standard field code
<div class="g-recaptcha" data-sitekey="--------------YOUR-PUBLIC-KEY------------"></div>
and it works well

**Comment by senseimknight** [comment]
If a client-side call to `grecaptcha.getResponse()` returns a correct response, then this workaround should help solve the problem: [Getting Null in g-recaptcha-response Google's reCaptcha](https://stackoverflow.com/questions/40078800/getting-null-in-g-recaptcha-response-googles-recaptcha/49739306)
